autocertkit/test_runner.py

```diff
@@ -237,6 +237,9 @@ def run_tests_from_file(test_file):
             log.debug("Reboot all hosts")
             reboot_all_hosts(session)
             try:
+                # Just wait host reboot and do not exit immediately,
+                # otherwise status.py will get wrong status then
+                time.sleep(300)
```

Comment:
I don't like the idea of putting in this sort of `sleep` - how can we prevent `status.py` getting the wrong status? Perhaps that needs to be updated to say that a reboot is happening?

Reply:
See [section 3 ofdesign](https://info.citrite.net/display/xenserver/REQ-46%3A+HCL+design+for+SR-IOV+pass-through+support#REQ-46:HCLdesignforSR-IOVpass-throughsupport-3.Frameworkimprovement(non-functionalrequirement))
As my understanding, 'reboot' is only a test step, not test status, test has just 3 status: init, running and done. ACK status is same, it has no reboot status. it will be easy to understand if ACK run on another separate machine (like xenrt controller). If ACK stopped it indicates error happening.

Assume host is rebooting but network service still alive, and ACK existed, then status.py executed by end user (or xenrt test case) in the narrow gap, status.py will report error that tests have not finished but ACK process is not running. Add "time.sleep(300)" will prevent the occurrence.

Also seeing ACK init.d script, it works as service, it won't/should not stop.
```
# vi /etc/rc3.d/S99auto-cert-kit
stop() {
#Nothing, do nothing
log "Stopping"
}
```